packages/visitor-keys/src/visitor-keys.ts

```diff
@@ -206,7 +206,8 @@ const additionalKeys: AdditionalKeys = {
   TSDeclareFunction: SharedVisitorKeys.Function,
   TSDeclareKeyword: [],
   TSEmptyBodyFunctionExpression: ['id', ...SharedVisitorKeys.FunctionType],
-  TSEnumDeclaration: ['id', 'members'],
+  TSEnumBody: ['members'],
+  TSEnumDeclaration: ['body', 'id'],
```

Comment:
```suggestion
  TSEnumDeclaration: ['id', 'body'],
```

These need to be placed in the order that they should be visited during a traversal.
EG the order they appear in source